src/test/java/com/blackducksoftware/integration/hub/alert/audit/repository/AuditEntryRepositoryWrapperTest.java

```diff
@@ -0,0 +1,48 @@
+package com.blackducksoftware.integration.hub.alert.audit.repository;
+
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import com.blackducksoftware.integration.exception.EncryptionException;
+import com.blackducksoftware.integration.hub.alert.OutputLogger;
+import com.blackducksoftware.integration.hub.alert.audit.mock.MockAuditEntryEntity;
+
+public class AuditEntryRepositoryWrapperTest {
+    private OutputLogger outputLogger;
+
+    @Before
+    public void init() throws IOException {
+        outputLogger = new OutputLogger();
+    }
+
+    @After
+    public void cleanup() throws IOException {
+        outputLogger.cleanup();
+    }
+
+    @Test
+    public void testFindFirstByCommonConfigIdOrderByTimeLastSentDesc() throws IOException {
```

Comment:
I think we want to test custom queries with a live database.

Reply:
If we already do this in the repository test, feel free to ignore. This seems fine for the wrapper test.

Reply:
These tests are for testing exceptions specifically, so a DB connection isn't required. I can try to find the accompanied IT if it exists and add it there, but right now this runs as a unit test as opposed to an IT